src/DashRenderer.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-env browser */
+
+'use strict';
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import AppProvider from './AppProvider.react';
+import PropTypes from 'prop-types';
+
+class DashRenderer {
+    constructor(hooks) {
+        // render Dash Renderer upon initialising!
+        ReactDOM.render(
+            <AppProvider hooks={hooks} />,
+            document.getElementById('react-entry-point')
+        );
+    }
+}
+
+DashRenderer.propTypes = {
```

Comment:
Oh, per @chriddyp 's earlier comment, `DashRenderer` isn't a react component so it doesn't have props. But `AppProvider` is a component, so this can go there.

Can we add a test that tries to send non-function hooks and show that it fails in an intelligible way?